go-controller/pkg/node/node.go

```diff
@@ -528,6 +528,12 @@ func (n *OvnNode) Start(ctx context.Context, wg *sync.WaitGroup) error {
 			defer wg.Done()
 			nodeController.Run(n.stopChan)
 		}()
+	} else {
+		// attempt to cleanup the possibly stale bridge
+		_, stderr, err := util.RunOVSVsctl("--may-exist", "del-br", "br-ext")
+		if err != nil {
+			klog.Errorf("Deletion of bridge br-ext failed: %v (%v)", err, stderr)
+		}
```

Comment:
@JacobTanenbaum do we also need to clean up the patch-port that gets created between br-ext and br-int?

Reply:
the patch port on the bridge `br-ext` gets cleaned up but not the`br-int` port (`int`). fixed to explicitly clean up